src/components/HeaderWithMenu.tsx

```diff
@@ -1,11 +1,12 @@
 import { StackNavigationProp } from '@react-navigation/stack'
```

Comment:
I think `HeaderWithMenu` is not good naming anymore, this is specific for the home screen, so imo `HomeScreenHeader` would be better.